engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/error/AssertionError.java

```diff
@@ -0,0 +1,18 @@
+package org.enso.interpreter.node.expression.builtin.error;
+
+import java.util.List;
+import org.enso.interpreter.dsl.BuiltinType;
+import org.enso.interpreter.node.expression.builtin.UniquelyConstructibleBuiltin;
+
+@BuiltinType
+public class AssertionError extends UniquelyConstructibleBuiltin {
```

Comment:
I'd add `final`.

Reply:
Done in 3fe75ab947dab777933a5482be08ca19a876856a. I always forget about the `final` classes ...